packages/message-encryption/src/ecies.ts

```diff
@@ -95,13 +97,15 @@ export interface EncoderOptions extends BaseEncoderOptions {
  * in [26/WAKU2-PAYLOAD](https://rfc.vac.dev/spec/26/).
  */
 export function createEncoder({
+  pubSubTopic = DefaultPubSubTopic,
   contentTopic,
   publicKey,
   sigPrivKey,
   ephemeral = false,
   metaSetter
 }: EncoderOptions): Encoder {
   return new Encoder(
+    pubSubTopic,
```

Comment:
I'd put this parameter last so it can be marked optional when we implement autosharding